File: src/main/java/net/hectus/neobb/turn/default_game/mob/TPhantom.java
package net.hectus.neobb.turn.default_game.mob;

import net.hectus.neobb.player.NeoPlayer;
import net.hectus.neobb.turn.default_game.attributes.clazz.SupernaturalClazz;
import net.hectus.neobb.turn.default_game.attributes.function.AttackFunction;
import org.bukkit.entity.Phantom;

public class TPhantom extends MobTurn<Phantom> implements AttackFunction, SupernaturalClazz {
    public TPhantom(NeoPlayer player) { super(player); }
    public TPhantom(Phantom data, NeoPlayer player) { super(data, player); }

    @Override
    public int cost() {
        return 5;
    }
}

File: src/main/kotlin/net/hectus/neobb/turn/default_game/mob/TPhantom.kt
package net.hectus.neobb.turn.default_game.mob

import com.marcpg.libpg.storing.Cord
import net.hectus.neobb.player.NeoPlayer
import net.hectus.neobb.turn.default_game.attribute.clazz.SupernaturalClazz
import net.hectus.neobb.turn.default_game.attribute.function.AttackFunction
import org.bukkit.entity.Phantom

class TPhantom(data: Phantom?, cord: Cord?, player: NeoPlayer?) : MobTurn<Phantom>(data, cord, player), AttackFunction, SupernaturalClazz {
    override val cost: Int = 5
}
